apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -729,14 +898,18 @@ export function buildMachine(
             },
             eject_to_front: {
               invoke: pollPaperStatus(),
-              entry: ['ejectPaperToFront'],
+              entry: [
+                'resetContext',
+                'endCardlessVoterAuth',
+                'ejectPaperToFront',
+              ],
               on: {
-                NO_PAPER_ANYWHERE: 'resetting_state_machine_after_success',
+                NO_PAPER_ANYWHERE: 'not_accepting_paper',
                 // Sometimes paper ejected to front is not held by the motors but
                 // will still trigger input sensors.
                 // In this case we still want to allow the machine to progress.
-                PAPER_READY_TO_LOAD: 'resetting_state_machine_after_success',
-                PAPER_IN_INPUT: 'resetting_state_machine_after_success',
+                PAPER_READY_TO_LOAD: 'not_accepting_paper',
+                PAPER_IN_INPUT: 'not_accepting_paper',
```

Comment:
Not directly related to this change, but I initially tried using the `eject_to_front` state for returning ballots when the new `returnPreprintedBallot()` is called and realised that puts the app into a state with the `"your ballot was cast"` message - since `eject_to_front` is only accessible from `not_accepting_paper` at the moment, changing this to return to `not_accepting_paper` after ejecting